### PR TITLE
Some temporary basic webhook logging

### PR DESCRIPTION
**Pull request by thermistor**

### CODE REVIEW

File: lib/payment_provider/handlers/async_handler.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ class AsyncHandler
       # 'customer.subscription.created' => PaymentProvider::Handlers::SubscriptionHandler,
 
       def call(event)
+        ::Rails::logger.error("WEBHOOK: #{event.type} CONNECT: #{event.try(:account)} LIVEMODE: #{event.livemode}")
```
Comment (rbarreca) [inline lib/payment_provider/handlers/async_handler.rb]: Reason it is error vs info (if there is such a level on Rails logger)?

Comment (thermistor) [reply lib/payment_provider/handlers/async_handler.rb]: There is such a log level, I didn't want it filtered, but info would also work on our prod. Normally prod log level would be higher.

File: lib/payment_provider/handlers/async_handler.rb [hunk 2]
```diff
@@ -11,6 +11,7 @@ class AsyncHandler
       # 'customer.subscription.created' => PaymentProvider::Handlers::SubscriptionHandler,
 
       def call(event)
+        ::Rails::logger.info("WEBHOOK: #{event.type} CONNECT: #{event.try(:account)} LIVEMODE: #{event.livemode}")
```
Comment (ivar) [inline lib/payment_provider/handlers/async_handler.rb]: info logs go to Rollbar ?

Comment (thermistor) [reply lib/payment_provider/handlers/async_handler.rb]: No, they go to PaperTrail, only exceptions go to Rollbar.

Comment (rbarreca) [reply lib/payment_provider/handlers/async_handler.rb]: `Rollbar.info()` et al go to Rollbar obviously. But @thermistor you're saying Rails logger.info does not? @ivar I see your comment says "yes".

Comment (thermistor) [reply lib/payment_provider/handlers/async_handler.rb]: Yes I am saying Rails::logger does not. Exceptions are sent to Rollbar automatically, and explicit calls to `Rollbar.notify('an exception or message')` of course also.

Comment (rbarreca) [reply lib/payment_provider/handlers/async_handler.rb]: Cool, guess I'm just wondering what @ivar 's "yes" is about?

**Pull request merged**